app/server.go

```diff
@@ -216,6 +216,8 @@ func NewServer(options ...Option) (*Server, error) {
 	mlog.Info(fmt.Sprintf("Current working directory is %v", pwd))
 	mlog.Info("Loaded config", mlog.String("source", s.configStore.String()))
 
+	s.PushHostChecker()
```

Comment:
I don't love the name, and I think it should be private to the server (nobody need to access to this from the outside).

Probably something like `checkPushNotificationServerUrl`, is more explicit about what the function does.